fix(autoscaler): switch from the deprecated beta version (v2beta2) to the GA version (v2)

gkalpak opened the pull request:
The previously used beta version of `HorizontalPodScaler`, `autoscaling/v2beta2`, has been deprecated and will be removed in Kubernetes v1.26.

Upgrade to the GA version, `autoscaling/v2`, (available since v1.23) to be ready for the v1.26 upgrade.

For more details see the [v1.26 deprecations guide][1] as well as xlts-dev/registry#304.

[1]: https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-26

Splaktar commented on charts/verdaccio/Chart.yaml:
```diff
@@ -1,7 +1,7 @@
 description: A lightweight, private NPM proxy registry for GKE
 name: verdaccio-gke-charts
-version: 0.2.8
+version: 0.3.8
```
```suggestion
version: 0.3.0
```

gkalpak committed:
fix(autoscaler): switch from the deprecated beta version (v2beta2) to the GA version (v2)

The previously used beta version of `HorizontalPodScaler`,
`autoscaling/v2beta2`, has been deprecated and will be removed in
Kubernetes v1.26.

Upgrade to the GA version, `autoscaling/v2`, (available since v1.23) to
be ready for the v1.26 upgrade.

For more details see the [v1.26 deprecations guide][1] as well as
xlts-dev/registry#304.

[1]: https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-26
